Anytype/Sources/PresentationLayer/Modules/SpaceCreate/SpaceCreateView.swift

```diff
@@ -30,5 +30,18 @@ struct SpaceCreateView: View {
         .onChange(of: model.dismiss) { _ in
             dismiss()
         }
+        .ignoreSafeAreaKeyboardLegacy()
+    }
+}
+
+private extension View {
+    // Fix glitch when user dismiss screen with opened keyboard
+    @available(iOS, deprecated: 16.4)
+    func ignoreSafeAreaKeyboardLegacy() -> some View {
+        if #available(iOS 16.4, *) {
+            return self
+        } else {
+            return self.ignoresSafeArea(.keyboard)
+        }
     }
```

Comment:
why do we need to make a condition here? `ignoresSafeArea(.keyboard)` works before 16.4 if i'mnot mistaken

Reply:
From ios 16.4 safe area for keyboard working normally. I disabled keyboard safe area to prevent glitch for ios 16.2

iOS 16.4

https://github.com/anyproto/anytype-swift/assets/2483784/82baa5bf-fb6c-4df4-973f-0714c41a8c76

iOS 16.2 glitch

https://github.com/anyproto/anytype-swift/assets/2483784/5ac5c217-5857-415b-9735-c1c6082fd456

Reply:
got it!